<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>run_setting_gui</class>
 <widget class="QDialog" name="run_setting_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="l_Title2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>쌀 헹굼 서비스</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_Pic2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_RiceQuantity">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>쌀 양:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_ManNum">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>216</y>
     <width>46</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_Man">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>인분</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRun">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>동작</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>118</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_State">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>380</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>진행중</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
